kratos/solving_strategies/schemes/residual_based_adjoint_bossak_scheme.h

```diff
@@ -83,6 +84,14 @@ class ResidualBasedAdjointBossakScheme : public Scheme<TSparseSpace, TDenseSpace
         mBossak.Alpha = Settings["alpha_bossak"].GetDouble();
     }
 
+    ResidualBasedAdjointBossakScheme(
+        const double BossakAlpha,
+        AdjointResponseFunction::Pointer pResponseFunction
+        ) : mpResponseFunction(pResponseFunction)
+    {
+        mBossak.Alpha = BossakAlpha;
+    }
+
```

Comment:
same, the `Parameters` based ctor already contains this so why adding this one?

Reply:
I'll remove it, I was going to use this in python level so, I don't have to create Kratos.Parameters object to create this bossak scheme. But I understand :)

Reply:
removed :)